skgstat/Variogram.py

```diff
@@ -1454,7 +1457,7 @@ def preprocessing(self, force=False):
         self._calc_diff(force=force)
         self._calc_groups(force=force)
 
-    def fit(self, force=False, method=None, sigma=None, **kwargs):
+    def fit(self, force=False, method=None, sigma=None, bounds=None, p0=None, **kwargs):
```

Comment:
we should document these two new attributes. Especially, because setting them to `None` will cause the function to actually use some defaults calculated from the sample. The underlying `curve_fit` function from scipy, will just not use bounds, if `None` is passed. This is a small inconsistency, that we should make clear.

Reply:
Done!